sklearn/linear_model/sgd_fast.pyx

```diff
@@ -319,7 +319,7 @@ cdef class SquaredEpsilonInsensitive(Regression):
         z = y - p
         if z > self.epsilon:
             return -2 * (z - self.epsilon)
-        elif z < self.epsilon:
+        elif z < -self.epsilon:
```

Comment:
Hi, it would be fine to add what's new entry and also a regression test if possible ( need to make up one as it is not being caught before) and push the new commits to the same branch (same PR). Hope that is the right thing to do here.